doc/developer/webservices.rst

```diff
@@ -240,3 +289,26 @@ Result
 
 The 'Content-Type' header is 'text/csv',
 and the data contains the given 'csv' data.
+
+Geometry operations
+===================
+
+This service gives geometry operations (actually only one)
+
+Difference
+----------
+
+URL: ``.../difference``
+
+Method: ``POST``
+
+Data:
+
+.. code:: json
+
+   {
+       "geom": <The geom to extrude>
+       "diff": <The geom to extrude with>
```

Comment:
But it's compliant with the PostGIS doc. http://postgis.net/docs/ST_Difference.html
Also, we often see schemas where A & B names are used.
![text4247](https://cloud.githubusercontent.com/assets/319774/9490512/9f6565ac-4be8-11e5-8ae7-47ea0d7807d3.png)

Even if not perfect, I find geomA and geomB easier to understand. I would be interested to know what others think.